README.md

```diff
@@ -3,7 +3,7 @@ Contributors: studiopress, nathanrice, bgardner, dreamwhisper, laurenmancke, sha
 Tags: social media, social networking, social profiles
 Requires at least: 4.0
 Tested up to: 6.0
-Stable tag: 3.1.0
+Stable tag: 3.1.1
```

Comment:
One thing I forgot to mention: could you add a Changelog entry at line 75 for version 3.1.1?